packages/core/content-manager/server/src/services/data-mapper.ts

```diff
@@ -62,14 +63,14 @@ const formatAttribute = (key: any, attribute: any) => {
 };
 
 // FIXME: not needed
-const toRelation = (attribute: any) => {
+const toRelation = (attribute: Attribute.Relation) => {
   return {
     ...attribute,
-    type: 'relation',
+    // @ts-expect-error target to found on Attribute.Relation
```

Comment:
@Marc-Roig do you know what type I should be using here?

Reply:
I am not sure, I believe Attribute.Relation should be the valid one 🤔  but maybe this is an attribute used frontend side.
Looks like the backend is mapping target to target model , so maybe we coul dhave this definition in the shared files.